README.md

```diff
@@ -3,148 +3,89 @@
 
     ("seek, and ye shall eat spinach" - My primary school teacher)
 
-This is a fast text search engine, intended for use with source
+Zoekt is a text search engine intended for use with source
 code. (Pronunciation: roughly as you would pronounce "zooked" in English)
 
 **Note:** This is a [Sourcegraph](https://github.com/sourcegraph/zoekt) fork
-of [github.com/google/zoekt](https://github.com/google/zoekt). It is now the
-main maintained source of Zoekt.
+of the original repository [github.com/google/zoekt](https://github.com/google/zoekt).
+It is now the maintained source for Zoekt.
```

Comment:
I would combine these two lines and have one line like

This primary source for Zoekt, actively maintained by [Sourcegraph](https://github.com/sourcegraph/zoekt) since 20??. Prior is was maintained at [github.com/google/zoekt](https://github.com/google/zoekt)